src/segment-loader.js

```diff
@@ -771,6 +775,11 @@ export default class SegmentLoader extends videojs.EventTarget {
    * operation is complete
    */
   remove(start, end, done = () => {}) {
+    // clamp end to duration if we need to remove everything
+    if (end === Infinity) {
```

Comment:
this needs to be clamped because of https://github.com/videojs/videojs-contrib-hls/pull/1225

Reply:
I would make this a comment so we can remove in the future!